time_manager/get_cal_time.F90

```diff
@@ -316,13 +317,15 @@ function get_cal_time(time_increment, units, calendar, permit_calendar_conversio
   increment_seconds = int(dt - increment_days*86400)
 else
   call error_mesg('get_cal_time','"'//trim(units)//'"'//' is not an acceptable units attribute of time.'// &
-    ' It must begin with: "years since", "months since", "days since", "hours since", "minutes since", or "seconds since"',FATAL)
+    ' It must begin with: "years since", "months since", "days since", "hours since", "minutes since", &
+            &  or "seconds since"',FATAL)
```

Comment:
This is inconsistent with how you have dealt with other strings in multi-line instances

Reply:
The script got messed up with this line actually so this is fixed to add the concatenation